buildifier/buildifier.go

```diff
@@ -113,6 +115,11 @@ func main() {
 		os.Exit(2)
 	}
 
+	if *tablesPath != "" && tables.ParseAndUpdateJsonDefinitions(*tablesPath) != nil {
```

Comment:
OK. Note that this undoes the original change where you asked me to combine the two conditions into a single `if`. I don't know if there's some clever way to assign `err` in the `if`, though perhaps even if that's possible, it's not considered good style, so I did the simple thing.